### DISCUSSION TITLE
Donations via Liberapay

### DISCUSSION THREAD

**Discussion by Currrupted** [zyrouge/symphony#332] [General]
Hello! I really like Symphony and would love to support the development financially. Would it be possible to donate using [Liberapay](https://liberapay.com)? That would make it easy to support you regularly, at least for me. Liberapay could also be useful to receive more attention, as they recommend projects and/or individuals who do not receive as much support yet.

**Comment by zyrouge** [comment]
Sadly, I will not be using Liberapay due to privacy reasons.
